MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.md LICENSE CODE_OF_CONDUCT.md CONTRIBUTING.md requirements.txt docs/*.md
+include LICENSE
```

Comment:
How is this different or not covered by the line above?

Reply:
Actually you are right. It is not different, I really missed the license string there.
I will recheck where from the tarball came off because based on the MANIFEST.in it should be there...